src/pages/index.html

```diff
@@ -91,22 +126,137 @@ <h2 class="field__title">Lĩnh vực kinh doanh</h2>
           <div class="circle circle--red circle--top-left"></div>
           <span class="box-tag">Digital Creative Studio</span>
         </div>
-        <div class="box-title">Creative<br>&amp; Engineering</div>
-        <div class="box-description">Đội ngũ nhân lực dồi dào chuyên về công nghệ, thiết kế và kinh doanh.</div>
-        <button class="learn-more-btn learn-more-btn--red">Tìm hiểu thêm →</button>
+        <div class="box-title">Creative<br />&amp; Engineering</div>
+        <div class="box-description">
+          Đội ngũ nhân lực dồi dào chuyên về công nghệ, thiết kế và kinh doanh.
+        </div>
+        <button class="learn-more-btn learn-more-btn--red">
+          Tìm hiểu thêm →
+        </button>
       </div>
-    
+
       <div class="box box--white">
         <div class="box-header">
           <div class="circle circle--white circle--top-left"></div>
           <span class="box-tag box-tag--light">Digital Creative Studio</span>
         </div>
-        <div class="box-title">Talent<br>Platform</div>
-        <div class="box-description">Các giải pháp nguồn nhân lực hỗ trợ tăng trưởng kinh doanh bền vững.</div>
-        <button class="learn-more-btn learn-more-btn--dark">Tìm hiểu thêm →</button>
+        <div class="box-title">Talent<br />Platform</div>
+        <div class="box-description">
+          Các giải pháp nguồn nhân lực hỗ trợ tăng trưởng kinh doanh bền vững.
+        </div>
+        <button class="learn-more-btn learn-more-btn--dark">
+          Tìm hiểu thêm →
+        </button>
         <div class="circle circle-right"></div>
         <div class="circle circle--down-right"></div>
       </div>
-    </section>    
+    </section>
+
+    <section class="project--container">
+      <div class="project--title">
+        <div class="circle project--down-left"></div>
+        <h2>Dự án của chúng tôi</h2>
+        <div class="circle project--top-right"></div>
+      </div>
+      <div class="project--content">
+        <div class="project--data">
+          <div class="project--items">
+            <div class="project--item">
+              <div class="project--number">200</div>
+              <div class="project--details">
+                <div class="project--plus">++</div>
+                <div class="project--label">Đối tác</div>
+              </div>
+            </div>
+            <div class="project--item">
+              <div class="project--number">300</div>
+              <div class="project--details">
+                <div class="project--plus">++</div>
+                <div class="project--label">Dịch vụ được hỗ trợ</div>
+              </div>
+            </div>
+          </div>
+          <button class="learn-more-btn learn-more-btn--dark">
+            Xem tất cả dự án →
+          </button>
+        </div>
+        <div class="project--slideshow">
```

Comment:
E mô tả luồng xử lý slides này giúp a nào

Reply:
Dạ luồng e xử lý slides này là :
- Có 3 input radio đại diện cho 3 slide
- Có 6 label tức mỗi slide có 2 nút chuyển trang pre next. Trạng thái ban đầu sẽ là display: none và khi input radio của một slide nào đó được chọn, các nút điều hướng tương ứng của slide đó sẽ hiển thị (display: block).
- Khi user nhấn chọn các nút chuyển trang (label) do label có thuộc tính for, chỉ định ID của input radio tương ứng nên sẽ thay đổi trạng thái của input radio ứng với mỗi slide
- Khi người dùng chọn một input radio, slide tương ứng sẽ được hiển thị bằng cách dịch chuyển container .slides. Các đoạn CSS như #slide1:checked ~ .slides điều chỉnh vị trí của .slides dựa trên trạng thái của các input radio.

Reply:
Dạ luồng e xử lý slides này là :
- Có 3 input radio đại diện cho 3 slide
- Có 6 label tức mỗi slide có 2 nút chuyển trang pre next. Trạng thái ban đầu sẽ là display: none và khi input radio của một slide nào đó được chọn, các nút điều hướng tương ứng của slide đó sẽ hiển thị (display: block).
- Khi user nhấn chọn các nút chuyển trang (label) do label có thuộc tính for, chỉ định ID của input radio tương ứng nên sẽ thay đổi trạng thái của input radio ứng với mỗi slide
- Khi người dùng chọn một input radio, slide tương ứng sẽ được hiển thị bằng cách dịch chuyển container .slides. Các đoạn CSS như #slide1:checked ~ .slides điều chỉnh vị trí của .slides dựa trên trạng thái của các input radio.